job-dsls/jobs/seed_job.groovy

```diff
@@ -52,7 +52,6 @@ job("a-seed-job") {
 
         jobDsl {
             targets("job-dsls/jobs/**/communityRelease_pipeline.groovy\n" +
-                    "job-dsls/jobs/**/deploy_development_version.groovy\n" +
```

Comment:
so we will not have this anymore? even it is in root directory after your remove it here it will be only on Jenkins instance, or am I wrong?

Reply:
this script belongs to the main root directory - not KIE/master/KIE/job - we still have it as DSL but it makes no sense in KIE/master - so it should be called in a-seed-job in KIE but not in the KIE/master/a-seed-job.

Reply:
ok, I got that you will create it in frame of https://issues.redhat.com/browse/BXMSPROD-1012